### ISSUE TITLE
Content: Linear Search

### ISSUE THREAD

**Issue by HarshCasper** [TesseractCoding/NeoAlgo-Docs#13]
## Description 

Add the article for `Linear Search`. The basic layout for the article should be: 

- What is Linear Search? 
- Explanation for Linear Search
- Algorithmic Steps 
- Code (Use C++/Java and Python; At least two languages should be present and pick up code from NeoAlgo with apt credits to author)
- Sample I/O 
- Time & Space Complexity 

This article should be between 500-700 words in length. You should use traditional Markdown to prepare the content and the use of external images is prohibited. You can however create your own images. Run a proper Grammarly Lint check before submitting the PR.

**Comment by SAY-droid427** [comment]
@HarshCasper I would like to work on this issue. Please assign it to me.

**Comment by HarshCasper** [comment]
Hi @SAY-droid427 

Thanks for taking this up! I hope you spend sometime in understanding the directory structure and where to make the PR. If you are confused, check it right here: https://github.com/TesseractCoding/NeoAlgo-Docs/tree/main/docs/searching-algorithms

I have already created a file for you named `linear-search.md`. And I have added the same on the sidebar. So I guess you will have a fair understanding of how to contribute (which is much different from NeoAlgo)
